macOS/DuckDuckGo/Localization/AppStore.xcstrings

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
This is empty for now so we could remove it.  That said we can also keep it in case we need AppStore specific strings.